djstripe/templatetags/djstripe_tags.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import division
 
 from django.template import Library
-
+from ..settings import DJSTRIPE_PLAN_HIERARCHY
```

Comment:
This should be (pretty sure it works):
``from .. import settings as djstripe_settings``